src/main/java/hospital/userservice/model/Department.java

```diff
@@ -0,0 +1,23 @@
+package hospital.userservice.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+public class Department {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true, nullable = false)
+    private UUID pbo;
```

Comment:
PBO stands for "Poslovni broj odeljenja" could you find some replacement for that abbreviation to be in english because everything else is already in english?